src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy

```diff
@@ -41,12 +42,20 @@ class GatlingPluginExtension {
             setBatchMode(batchMode)
         }
 
-        def setUrl(String url) {
-            this.url = url
+        def setApiUrl(String apiUrl) {
+            this.apiUrl = apiUrl
         }
 
-        def url(String url) {
-            setUrl(url)
+        def apiUrl(String apiUrl) {
+            setApiUrl(apiUrl)
+        }
+
+        def setWebAppUrl(String webAppUrl) {
+            this.webAppUrl = webAppUrl
+        }
+
+        def webAppUrl(String webAppUrl) {
+            setWebAppUrl(webAppUrl)
         }
```

Comment:
Not backwards-compatible (changes the name used in the build.gradle file), but this setting was never documented for end users.